java-manta-client-kryo-serialization/src/main/java/com/joyent/manta/serialization/CipherSerializer.java

```diff
@@ -178,6 +178,13 @@ private void registerWithMagicInstantiator(final Kryo kryo, final String classNa
     @Override
     @SuppressWarnings("unchecked")
     public void write(final Kryo kryo, final Output output, final Cipher object) {
+        if (object.getProvider() != null && !"BC".equals(object.getProvider().getName())) {
```

Comment:
Is there a (maybe still package private) constant that can be compared to instead?

Reply:
Yes, there sure is. Switching to using `BouncyCastleProvider.PROVIDER_NAME`.